pkg/chunk/aws_storage_client.go

```diff
@@ -47,7 +47,7 @@ const (
 var backoffConfig = util.BackoffConfig{
 	// Backoff for dynamoDB requests, to match AWS lib - see:
 	// https://github.com/aws/aws-sdk-go/blob/master/service/dynamodb/customizations.go
-	MinBackoff: 50 * time.Millisecond,
+	MinBackoff: 200 * time.Millisecond,
```

Comment:
The 50 was based on the comment above - the aws lib backs off 2^n*50ms. Since we are randomising between 0 and MinBackoff, I reckon the figure should be 100, so that the _mean_ is 50.

Reply:
That would be correct if we wanted to stay the same. As I said, I felt the old value was too low.

However it would be safer to do these two changes separately, to avoid confusing the effects.

Reply:
Changed to 100ms